packages/sdk/client/src/packlets/shell/style.css

```diff
@@ -0,0 +1,9 @@
+.__DXOS_SHELL {
+  position: fixed;
```

Comment:
i _think_ absolute is more precise (with left,right,top,bottom = 0); possibly add flex so that children can avail themselves of that.

Reply:
using this method I found that the iframe didn't actually get fullscreened, I couldn't tell you exactly why but I'm going to get @thure to help with some of the css for the shell as we'll want to add some transitions and other elegance